lib/clients/http_client.js

```diff
@@ -628,6 +629,15 @@ HttpClient.prototype._options = function (method, options) {
         opts.socketPath = this.socketPath;
     }
 
+    // Add the request path to the base path of the client
+    // Don't combine the request url and client path if they both
+    // have a slash
+    if (opts.path && this.url.path &&
+        opts.path !== '/' && this.url.path !== '/') {
+        // clean up the url path, stripping the last /
+        opts.path = utils.sanitizePath([this.url.path, opts.path].join('/'));
```

Comment:
That's a bummer, maybe we should look into something like URIjs, there's a lot of URL mugging that should be centralized (plugins too).

Re: above sorry, I should have been clearer, I just think that we should be consistent, presumably if users need strict URLs on the server, them they need a client that can hit those strict URLs. If we strip trailing slashes automatically them it's impossible to do so. Maybe we can do a strict:true option?